index.js

```diff
@@ -162,6 +164,87 @@ const inventoryCommand = (message) => {
   });
 };
 
+const lookUpCommand = (message) => {
+  const originalMessage = message.content;
+  const fuzzySearchKey = originalMessage.slice(8);
+
+  axios
+    .get(
+      "https://db.ygoprodeck.com/api/v7/cardinfo.php?&fname=" + fuzzySearchKey
+    )
+    .then(function (cardSearchResults) {
+      console.log(
```

Comment:
this is a good `console.log`, might be best done before the axios itself so that even when the axios fails, we'll see what URL was hit in the logs